src/tests/api/BavHappyPath.test.ts

```diff
@@ -86,22 +86,20 @@ describe("BAV CRI happy path tests", () => {
 			validateTxMAEventData({ eventName: "BAV_COP_REQUEST_SENT", schemaName: "BAV_COP_REQUEST_SENT_SCHEMA" }, allTxmaEventBodies);
 			validateTxMAEventData({ eventName: "BAV_COP_RESPONSE_RECEIVED", schemaName: "BAV_COP_RESPONSE_RECEIVED_SCHEMA" }, allTxmaEventBodies);
 		});
-
 		it.each([
-			{ firstName: "Ashley", lastName: "Allen" },
-			{ firstName: "Deborah", lastName: "Dawson" },
-			{ firstName: "Nigel", lastName: "Newton" },
-			{ firstName: "Yasmine", lastName: "Dawson" },
-			{ firstName: "Yasmine", lastName: "Newton" },
-			{ firstName: "Yasmine", lastName: "Palmer" },
-		])("Name Retry Test for $firstName $lastName", async ({ firstName, lastName }: { firstName: string; lastName: string }) => {
-			const newBavStubPayload = structuredClone(bavStubPayload);
-			newBavStubPayload.shared_claims.name[0].nameParts[0].value = firstName;
-			newBavStubPayload.shared_claims.name[0].nameParts[1].value = lastName;
+			["00111112"],
+			["00111113"],
+			["00111114"],
+			["22222222"],
+			["33333333"],
+			["44444444"],
+		])("Name Retry Test for Account Number: $accountNumber", async (accountNumber: string) => {
+			const newVerifyAccountYesPayload = structuredClone(verifyAccountYesPayload);
+			newVerifyAccountYesPayload.account_number = accountNumber;
 
-			const sessionId = await startStubServiceAndReturnSessionId(newBavStubPayload);
+			const sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
```

Comment:
Do we even need a stub payload at all now?

Reply:
Actually just started removing the stub payload references and realised removing this entirely will make it very difficult to test the /person-info endpoint.

<img width="1050" alt="image" src="https://github.com/govuk-one-login/ipv-cri-bav-api/assets/110032361/1811c163-2206-4be8-88f5-fb5d0c68cce8">

Reply:
Removed from all tests apart from /person-info enpoint